cogs/release.py

```diff
@@ -0,0 +1,198 @@
+#un implemented
+
+from asyncio.windows_events import NULL
+import discord
+from discord import client
+from discord.ext.commands import Cog, command, has_any_role
+from discord.ext import commands
+from utils import load_config
+
+global_config = load_config()
+config = global_config
+
+# TWITTER KEYS (...would go here)
+consumer_key = None
```

Comment:
should put these in the config.yml file

Reply:
done. it didn't get pushed to this version for some reason.